Makefile

```diff
@@ -17,6 +17,7 @@ EMBED_CONTAINER_IMAGES ?= 0
 # Options used in the 'run' target
 LVM_VOLSIZE ?= 1G
 ISOLATED_NETWORK ?= 0
+EXPOSE_KUBEAPI_PORT ?= 0
```

Comment:
Let's set this to 1 as a default to allow for a better user experience when running `make env`. We should give a clear message if the port is taken when `make run`